DfE.FindInformationAcademiesTrusts/appsettings.json

```diff
@@ -15,6 +15,18 @@
     "RedirectUri": "",
     "CallbackPath": "/signin-oidc"
   },
+  "DfE.FindInformationAcademiesTrusts.API": {
```

Comment:
Do we definitely need all of these to call the API? I'm wondering whether we can get away with just the `BaseUri` and `ClientSecret`

Reply:
We need the AzureAd config so that we can go over to AD as the Web App to acquire a Bearer Token, which we then pass to the API as an authorised credential